Despite having a correct checksum, I see it is failing to verify the checksum in GHA:
```
Run . spack-core/share/spack/setup-env.sh
==> Error: Package tau does not have a valid URL for version 2.35.1
Error: Process completed with exit code 3.
```

--- a/repos/spack_repo/builtin/packages/tau/package.py
+++ b/repos/spack_repo/builtin/packages/tau/package.py
@@ -19,7 +19,7 @@ class Tau(Package):
     Java, Python.
     """
 
-    maintainers("wspear", "eugeneswalker", "khuck", "sameershende")
+    maintainers("wspear", "eugeneswalker", "sameershende")
     homepage = "https://www.cs.uoregon.edu/research/tau"
     url = "https://www.cs.uoregon.edu/research/tau/tau_releases/tau-2.30.tar.gz"
     git = "https://github.com/UO-OACISS/tau2"

--- a/repos/spack_repo/builtin/packages/tau/package.py
+++ b/repos/spack_repo/builtin/packages/tau/package.py
@@ -29,6 +29,7 @@ class Tau(Package):
     license("MIT")
 
     version("master", branch="master")
+    version("2.35.1", sha256="6eadcf9e2a267d75d7dcae07cf24ab5975576de7ebd6eacb43e65a375eb8468e")
     version("2.35", sha256="b13c6a0579da59853f8e6482d5f3aaed482bc1306c4eb91411c1568f647bf348")
     version("2.34.1", sha256="0e90726372fa1b6f726eb62b0840350070a00215144853ee07a852a99458c619")
     version("2.34", sha256="229ab425e0532e635a0be76d60b8aa613adf7596d15a9ced0b87e7f243bb2132")

--- a/repos/spack_repo/builtin/packages/tau/package.py
+++ b/repos/spack_repo/builtin/packages/tau/package.py
@@ -66,6 +66,12 @@ class Tau(Package):
     darwin_default = not _is_darwin
     libunwind_darwin_default = "none" if _is_darwin else "shared"
 
+    variant(
+        "julia",
+        default=False,
+        description="Activate Julia support",
+        when="@2.35.1:",
+    )
     variant("scorep", default=False, description="Activates SCOREP support")
     variant("openmp", default=False, description="Use OpenMP threads")
     variant("pthreads", default=True, description="Use POSIX threads")
@@ -212,6 +218,9 @@ class Tau(Package):
     depends_on("java", type="run")  # for paraprof
     depends_on("oneapi-level-zero", when="+level_zero")
     depends_on("dyninst@12.3.0:", when="+dyninst")
+    depends_on("julia@1.6:", when="+julia")
+
+    conflicts("+julia", when="~pthreads ~ittnotify", msg="Julia support requires +pthreads and +ittnotify")
 
     conflicts("+comm", when="@:2.34 +python", msg="Bug in +comm with +python up to @2.34")
 

